bitnode.yml

```diff
@@ -41,31 +41,3 @@
         int_link: br0
         int_ipv4: "{{ host_database.bitnode.interfaces.br0.ipv4 }}"
         int_ipv6: "{{ host_database.bitnode.interfaces.br0.ipv6 }}"
-
```

Comment:
Vi gjør ingenting for å rydde opp automatisk, men det er kanskje greit, dette er mer for å sette opp på nytt i tilfelle rottefelle?

Reply:
Mja, i praksis så har vi tatt ting som skal bort for hånd og så justert ansible i etterkant. Vi er mere på pets enn cattle for all overskuelig fremtid.